### ISSUE TITLE
Docker Service : rsync hangs at end of transfer?

### ISSUE THREAD

**Issue by JacobSanford** [travis-ci/travis-ci#4899]
I've been experiencing a new issue whilst using the Docker service where a  rsync command inside the container hangs once the transfer is complete. Is this a known issue? I've tinkered a bit with verbosity and whatnot to determine what is happening, with little luck. I can't replicate this locally at all. 

Command that Freezes : https://github.com/unb-libraries/docker-drupal/blob/apache/init/60_build_drupal_tree.sh#L13
TravisFile: https://github.com/unb-libraries/docker-drupal/blob/apache/.travis.yml
Log : https://api.travis-ci.org/jobs/84284646/log.txt?deansi=true
